apps/openmw/mwrender/effectmanager.cpp

```diff
@@ -26,6 +26,13 @@ EffectManager::~EffectManager()
 }
 
 void EffectManager::addEffect(const std::string &model, const std::string& textureOverride, const osg::Vec3f &worldPosition, float scale, bool isMagicVFX)
+{
+    osg::Vec3f scale3f (scale, scale, scale);
+
+    addEffect(model, textureOverride, worldPosition, scale3f, isMagicVFX);
```

Comment:
Why add the overload for non-uniform scales when it's not being used (as far as I can tell)?